docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -1187,6 +1187,30 @@ public class KubernetesServerTestResource
 }
 ----
 
+If you want to make the annotation repeatable, the containing annotation type must be annotated with `@QuarkusTestResourceRepeatable`.
+For example, this would define a repeatable `@WithRepeatableTestResource` annotation.
+
+[source,java]
+----
+@QuarkusTestResource(KubernetesServerTestResource.class)
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+@Repeatable(WithRepeatableTestResource.List.class)
+public @interface WithRepeatableTestResource {
+
+    String key() default "";
+
+    @Target(ElementType.TYPE)
+    @Retention(RetentionPolicy.RUNTIME)
+    @QuarkusTestResourceRepeatable(WithRepeatableTestResource.class)
```

Comment:
I'm a bit late to the party but can you tell me why you need this particular annotation? I would expect us to be able to handle repeatable annotations without an additional annotation.

Reply:
Repeatable annotations are wrapped in a containing annotations and there is no base type to figure out it is a containing annotations with a value of type array with the actual annotations.
This would imply to parse all annotations with lots of reflection code. Nothing impossible I guess but the current loop would need to be changed. Would you like a proposal PR ?

Reply:
@scrocquesel isn't it a good use case for `IndexView#getAnnotationsWithRepeatable()`. I might miss something but I think we could leverage Jandex to do the work for us. And if not possible, we might want to talk to @Ladicek to make it possible.

Reply:
I don't really know what this PR is about, but seeing that it targets the `test-framework` module, I'm pretty sure Jandex isn't helpful there. This is all JUnit and runtime and reflection.

Reply:
Ah so that was the thing I was missing. Thanks for clarifying @Ladicek :).